Discord.Net.Abstractions/Rest/IDiscordRestClient.cs

```diff
@@ -120,6 +120,7 @@ public async Task<IRestGuild> GetGuildAsync(ulong id, RequestOptions options = n
                 ?.Abstract();
 
         /// <inheritdoc />
+        [Obsolete("This endpoint is deprecated, please use GetGuildWidgetAsync instead.")]
```

Comment:
This matches an annotation added by Discord.NET?

Reply:
correct. this matches an annotation added by discord.net